doc/usage.md

```diff
@@ -365,6 +365,9 @@ cl_active
 cl_waiting
 :   Client connections that have sent queries but have not yet got a server connection.
 
+cl_cancel_req
+:   Client connections that have sent query cancellations.
```

Comment:
I think this description is a bit misleading. It seems to imply that it is an all-time count of cancel requests that have passed through the pool. But it's actually just the count of cancel requests that have not been forwarded to the server yet, similar to `cl_waiting`. Let's find a better description.